lib/auth/machineid/workloadidentityv1/decision.go

```diff
@@ -118,6 +120,19 @@ func decide(
 		dst.OrganizationalUnit = templated
 	}
 
+	ec := wi.GetSpec().GetSpiffe().GetJwt().GetExtraClaims()
+	if ec != nil {
```

Comment:
```suggestion
	if ec := wi.GetSpec().GetSpiffe().GetJwt().GetExtraClaims(); ec != nil {
```

nit: just because I am lazy